src/hooks/useKnockoutExplorer.ts

```diff
@@ -237,29 +238,7 @@ async function configurePortal(explorerParams: ExplorerParams): Promise<Explorer
             inputs.extensionEndpoint = configContext.PROXY_PATH;
```

Comment:
I think this if statement is already included in `updateContextsFromPortalMessage`. We should be able to delete it.